_posts/2020-03-18-cockpit-215.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Cockpit 215
+author: kkoukiou
+date: 2020-03-18
+tags: cockpit linux
+slug: cockpit-215
+category: release
+summary: Cockpit with Network improvements and Docker deprecation
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 215.
+
+### Networking: Show additional ports for each firewall zone
+
+The Firewall page now shows ports that are directly added to a zone right after the services list.
+![zone ports](/images/firewall-zone-ports.png)
+
+### Docker: Deprecate in Ubuntu and Debian developement series
+
+`cockpit-docker` is being deprecated in favor of `cockpit-podman`, and not
+maintained upstream any more. We will stop building it for Ubuntu ≥ 20.04,
+Debian ≥ 11 (i. e. the current development series) and Fedora >= 32, but keep it for the
```

Comment:
Fedora 32 is also a development series, so please move the parenthesis after that.